src/App.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
+import Sum from './components/Sum';
 import './App.scss';
 
 const App = () => (
-  <h1>React sum</h1>
+  <>
+    <h1>React sum</h1>
+    <Sum a={5} b={-5} />
```

Comment:
Тут ошибка, в условии задания сказано что компонент `Sum` должен возвращаться 3 раза, и каждый раз нужно передавать разные пропсы: 2 и 2, 10 и 0, -5 и 5. 
> App component should render 3 Sum components for 2 + 2, 10 + 0 and -5 + 5

```suggestion
    <Sum a={2} b={2} />
    <Sum a={10} b={0} />
    <Sum a={5} b={-5} />
```
Вот так примерно.